### PR TITLE
QoL changes

### PR DESCRIPTION
**Pull request by Kanjirito**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -171,5 +171,17 @@
...
         }
     ])
-
+    
 })(angular, window, document)
+
+// When the user clicks on the button, scroll to the top of the document
+function topFunction() {
+  document.body.scrollIntoView(true) // For Safari
+  document.documentElement.scrollIntoView(true) // For Chrome, Firefox, IE and Opera
```
Comment (altbdoor) [inline script.js]: Hmm, `document.body.scrollIntoView()` works for me in Chrome and Firefox. Any reason why we use `document.documentElement` instead?

File: style.css [hunk 2]
```diff
@@ -130,8 +133,18 @@ body {
...
 
-
 /* Centers some stuff*/
 .ng-table table th{
     vertical-align: middle;
 }
+
+/* Scroll buttons style*/
+.scroll-btns-grp  {
+  position: fixed; /* Fixed position */
```
Comment (altbdoor) [inline style.css]: You can use the CSS class from Bootstrap instead 👍 
https://getbootstrap.com/docs/4.3/utilities/position/#common-values

File: style.css [hunk 3]
```diff
@@ -130,8 +133,18 @@ body {
...
+  bottom: 10px; /* Place the button at the bottom of the page */
+  right: 20px; /* Place the button 20px from the right */
+}
+
+.scroll-btns-grp .btn{
+    color: #000;
```
Comment (altbdoor) [inline style.css:148]: I prefer to leave it at their default color instead of overriding them 🤔

File: index.html [hunk 4]
```diff
@@ -229,9 +229,13 @@
             <li class="list-inline-item"><a href="http://ng-table.com/">ngTable</a></li>
         </ul>
     </div>
-
     <script src="https://cdn.jsdelivr.net/npm/angular@1.7.8/angular.min.js"></script>
     <script src="https://cdn.jsdelivr.net/npm/ng-table@3.0.1/bundles/ng-table.min.js"></script>
     <script src="./script.js"></script>
+    
+    <div class="btn-group btn-group-sm scroll-btns-grp" role="group" aria-label="Scroll buttons">
+      <button type="button" onclick=topFunction() title="Go to the top" class="btn page-link">▲</a>
```
Comment (altbdoor) [inline index.html]: We should wrap the `onclick` with double quotes. E.g., `onclick="topFunction()"`.

And personally, I don't use the `role` and `aria-*` attributes. Should be safe to remove them.

**Review by altbdoor — CHANGES_REQUESTED** [review]
Great stuff 👍 

Please consider the comments, thanks! 🤝

File: style.css (hunk 2 above)
Comment (Kanjirito) [reply style.css]: Changed

File: index.html (hunk 4 above)
Comment (Kanjirito) [reply index.html]: Done.

File: script.js (hunk 1 above)
Comment (Kanjirito) [reply script.js]: Not really.

File: style.css (hunk 3 above)
Comment (Kanjirito) [reply style.css:148]: You prefer the buttons being white? I think they look better in black.

Comment (altbdoor) [reply style.css:148]: Hmm, well we're not exactly UI/UX experts 😆 Its alright, this will do 👍

Comment (altbdoor) [reply style.css:148]: Hmm, well we're not exactly UI/UX experts 😆 Its alright, this will do 👍

**Pull request merged**